csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/EnvironmentVariableNames.cs

```diff
@@ -0,0 +1,8 @@
+namespace Semmle.Extraction.CSharp.DependencyFetching
+{
+    internal class EnvironmentVariableNames
```

Comment:
Yes, this is a good idea!
Maybe consider to write a small comment above each const, what the purpose of the environment variable is.